client/src/controllers/DropdownController.ts

```diff
@@ -162,50 +146,90 @@ export class DropdownController extends Controller<HTMLElement> {
       });
     }
 
+    const onCreate = (instance: Instance<Props>) => {
```

Comment:
What's the possibility of adding unit tests, we have a pretty solid baseline of unit tests for most other Stimulus controllers now.

It might be nice to see that continue, especially for additional core functionality like this.

I am happy to help with a rough version of a unit test for this if you need, I know I had a bit of trouble getting the Jest tests working for the Tippy integration on top of the initial development.

Reply:
It is very nice to see it continue, and I'd really love us to have tests for this, but it takes me quite a lot of time to write frontend tests so I thought I'd have the fix first just to unblock things. I do have a branch where I'm working on adding more tests. Not this one specifically yet, but ones like the `reset` value I added to `TeleportController`. I'll put up a PR for it soon.

Reply:
Ok. No problems. Yes it does take time.

I was thinking of doing a code coverage check (just a temp install , check and report) if the Stimulus controller codebase in a month or so. From there we could see if there are glaring gaps

Maybe raise some issues for contributors to help fill in anything that we deem important.